chainer/gradient_check.py

```diff
@@ -355,8 +355,14 @@ def check_backward(
     If :math:`r` is chosen from uniform distribution, we can conclude with
     high probability that the gradient of :math:`f` itself is correct.
 
-    If input objects (``x1_data`` or/and ``x2_data`` in this example) represent
-    integer variables, their gradients are ignored.
+    If the function is non-differentiable with respect to some input objects,
+    we can check its backprop to such objects by ``no_grads`` argument.
+    ``gradient_check`` computes numerical backward to inputs that correspond to
+    ``False`` in ``no_grads``. It also asserts the backprop lefts gradient
```

Comment:
Could you rephrase this as `... asserts that the backprop leaves gradients as None for inputs ...` ?